src/components/body/Body.tsx

```diff
@@ -10,10 +10,10 @@ type BodyProps = {
 const Body = ({onLanguageSwitch}: BodyProps) => {
   return (
     <div className="body-page">
-      <Header/>
+      <Header onLanguageSwitch={onLanguageSwitch}/>
```

Comment:
Please, insert a space before closing "/" at the end:
Header onLanguageSwitch={onLanguageSwitch} /